tools/mac_setup.sh

```diff
@@ -29,6 +31,8 @@ brew "openssl"
 brew "pyenv"
 brew "qt@5"
 brew "zeromq"
+brew "protobuf"
+brew "protobuf-c"
```

Comment:
and these?

Reply:
building casadi

Reply:
can you leave a comment so we can get rid of this once we can pip install casadi?